documentation/SRS.md

```diff
@@ -1,3 +1,185 @@
-#Manager for department and employees
+# Manager for department and employees
 
-*coming soon*
\ No newline at end of file
+## Vision
+
+This application is web-application that helps 
+navigate in information about employees and 
+departments. 
+
+Application should provide:
+* Storing information about employees and
+departments in database;
+* Display list of employees in department;
+* Updating employees data (add,editing,delete);
+* Display average cost of employees in a department;
+* Display list of departments;
+* Updating departments data (add,editing,delete);
+* Filtering by date of birthday and period of dates.
+
+## 1. Employees
+### 1.1 Display list about employees
+The mode is designed to view list of employees in department.
+When user selects a department, a list of employees for that 
+department appears **(pic 1.1)**.
+
+#### *Main scenario*
+* User select item Employees;
+* Application open employees list.
+
+![list_employees](./search_period.png)
+**pic 1.1 - View the employees list**
+
+The list displays the following columns:
+* ID - identification employee number 
+* Name - first and last employee name
+* Date of birthday - employee's date of birthday
+* Salary - employee salary
+### 1.2 Search by date and period
+For date search user have 2 mods.
+
+#### *Main scenario*
+1. Search by birthday:
+   
+   * User must uncheck **Period** and writes date
+   * Click search and list will appear (pic 1.2).
+   
+![search_date](./search_date.png)
+**pic 1.2 - View the employees list by date**
+
+2. Search by date period.
+
+   * User must check **Period** and writes dates. 
+   * Then click search and list will appear (pic 1.3).
+   
+![search_period](./list_employees.png)
+**pic 1.3 - View the employees list by period**
+
+For dropping date and period lists user can click button **Drop**.
+
+### 1.3 Add employee
+
+#### *Main scenario*
+* User clicks button **Add**;
+* Application displays form for adding;
+* User enters information and presses **Save**;
+* If all information is valid, then information is adding in DB;
+* If don't all information is valid, then error message is displaying.
```

Comment:
It would be great to see a mockup with an error message.

Reply:
Done